experimental/offline_attestation/README.md

```diff
@@ -0,0 +1,110 @@
+# Offline Attestation
+
+The current Oak Functions Runtime implementation uses an interactive handshake
+for setting up an attested end-to-end encrypted channel between the client and
+the runtime. For some scenarios it might be preferable to avoid the handshake.
+
+The offline attestation approach splits the attestation process into two
+distinct parts:
+
+- Key generation and publication: A single asymmetric key-pair is generated. The
+  public key is bound to an attestation report that provides evidence about the
+  enclave that generated the private key. The public key and attestation report
+  are then made availble to clients.
+- Encrypted communications: Clients use the public key (once they are satisfied
+  by the evidence that only appropriate enclaves would have access to the
+  private key) to encrypt requests to the server. Clients also provide
+  additional information (e.g. the client's public key) in the request to allow
+  the server to encrypt the response.
+
+Since the same public key is used for encrypting all requests this approach does
+not have the same forward secrecy that is provided by ephemeral keys in the
+interactive handshake. Avoiding the need for a handshake has some advantages
+though:
+
+- Reduced bandwith and latency
+- No need for sticky sessions in a load-balanced environment
```

Comment:
I would point out the downsides too, namely lack of replay protection and forward secrecy. WDYT?